src/Misc/layoutbin/installdependencies.sh

```diff
@@ -15,10 +15,17 @@ fi
 function print_errormessage() 
 {
     echo "Can't install dotnet core dependencies."
-    echo "You can manually install all required dependencies base on follwoing documentation"
+    echo "You can manually install all required dependencies based on following documentation"
     echo "https://docs.microsoft.com/en-us/dotnet/core/linux-prerequisites?tabs=netcore2x"
 }
 
+function print_rhel6message() 
+{
+    echo "Can't install dotnet core dependencies."
+    echo "You can manually install all required dependencies based on following documentation"
+    echo "https://docs.microsoft.com/en-us/dotnet/core/linux-prerequisites?tabs=rhel6"
```

Comment:
ToDo: add the documentation.

Reply:
https://github.com/dotnet/core/blob/master/Documentation/build-and-install-rhel6-prerequisites.md